### PR TITLE
tool: copy files from R2 to datalake

### PR DESCRIPTION
**Pull request by aonnikov**
- add tool for copying files to datalake
- add multipart upload option

### CODE REVIEW

File: packages/presentation/src/file.ts [hunk 1]
```diff
@@ -258,8 +258,8 @@ async function uploadFileWithSignedUrl (file: File, uuid: string, uploadUrl: str
       method: 'PUT',
       headers: {
         'Content-Type': file.type,
-        'Content-Length': file.size.toString(),
-        'x-amz-meta-last-modified': file.lastModified.toString()
+        'Content-Length': file.size.toString()
+        // 'x-amz-meta-last-modified': file.lastModified.toString()
```
Comment (haiodo) [inline packages/presentation/src/file.ts:262]: Do we need this change?

File: server/datalake/src/client.ts [hunk 2]
```diff
@@ -236,8 +274,8 @@ export class Client {
         method: 'PUT',
         headers: {
           'Content-Type': metadata.type,
-          'Content-Length': metadata.size?.toString() ?? '0',
-          'x-amz-meta-last-modified': metadata.lastModified.toString()
+          'Content-Length': metadata.size?.toString() ?? '0'
+          // 'x-amz-meta-last-modified': metadata.lastModified.toString()
```
Comment (haiodo) [inline server/datalake/src/client.ts:278]: same?

**Review by haiodo — APPROVED** [review]

File: packages/presentation/src/file.ts (hunk 1 above)
Comment (aonnikov) [reply packages/presentation/src/file.ts:262]: yes, this temporary fixes problem with presigned URL upload

File: server/datalake/src/client.ts (hunk 2 above)
Comment (aonnikov) [reply server/datalake/src/client.ts:278]: same

**Pull request merged**